versioned_docs/version-latest/farming-&-staking/farming/advanced-cli/platforms/_macos.mdx

```diff
@@ -58,6 +58,10 @@ After this, simply repeat the step you prompted for (step 4 or 6). This time, cl
 Using **run**, setting **--base-path** and specifying **--chain** became mandatory starting with Gemini 3h. The parameter `--validator` was switched to `--farmer` at the same time.
 :::
 
+:::note
+As of the Jun 11, 2024 version a new option is available for fast syncing called Snap Sync. It will only work for the initial sync, but many people report that they are fully synced within hours, not days.  The parameter is **--sync snap**
```

Comment:
```suggestion
As of the Jun 11, 2024 version a new option is available for fast syncing called Snap Sync](../cli-tips.mdx#Snap Sync). It will only work for the initial sync, but many people report that they are fully synced within hours, not days. The parameter is **--sync snap**.
```